addons/l10n_in/models/res_partner.py

```diff
@@ -28,6 +28,30 @@ class ResPartner(models.Model):
     )
 
     display_pan_warning = fields.Boolean(string="Display pan warning", compute="_compute_display_pan_warning")
+    l10n_in_gst_state_warning = fields.Char(compute="_compute_l10n_in_gst_state_warning")
```

Comment:
In 17.3 we should probably use `actionable_error` widget instead
https://github.com/odoo/odoo/commit/571fd206d1a808c3736863b3e30a3176e4f5cc6b

Maybe to update during the forward port? (to see if it would need an upgrade script)